bin/plugin/commands/release.js

```diff
@@ -189,7 +146,6 @@ async function runReleaseBranchCheckoutStep(
 
 	return {
 		version,
```

Comment:
Let's drop `version` from the returned object -- we're expecting it as a function arg now and aren't modifying it, so there's no point really to return it.

Reply:
You're right, I removed versionLabel for this reason and this slipped through the cracks.

Reply:
Thanks for fixing!